### ISSUE TITLE
Navigation: Docked version on small screens

### ISSUE THREAD

**Issue by edewit** [patternfly/patternfly-design#1243]
**Is this a new component or an extension of an existing one?**
What is the existing component, if any?
PageSidebar

**Describe the feature**
ATM on small screens the PageSidebar is not shown only when the hamburger is clicked it is revealed. Now it takes 2 clicks to navigate each time. What about a more minimalist  version instead of totally hiding it.

**Are there visuals for this feature? If applicable, please include examples for each state and for varying widths**
Something like this:
![image](https://user-images.githubusercontent.com/51133/222151898-18f2fd0d-4df2-44f3-9fd6-0fc3cd8c519c.png)

**Comment by andrew-ronaldson** [comment]
We're working on adding icons to top level navigation as a starting point. I think we need to spend time thinking about all the use cases to support this change and ensuring accessibility. Given our current priorities I'm moving this to the backlog until we have more time/resources to do this properly.

**Comment by bekah-stephens** [comment]
> We're working on adding icons to top level navigation as a starting point. I think we need to spend time thinking about all the use cases to support this change and ensuring accessibility. Given our current priorities I'm moving this to the backlog until we have more time/resources to do this properly.

I don't think there's really a way to do this without icons on each navigation item. What's the best way to start exploring this docked version? Do we just mock up an example using placeholder icons to visualize what it could look like if the use case comes up?

**Comment by andrew-ronaldson** [comment]
Yes this option should only apply if the consumer can provide icons for each nav item. If you can mock up using placeholder icons that would be a great start. 
We'll need to reflect nav item hover and selected states in this mock. Currently, our default nav uses subtle text/icon colours by default an regular on selected.

**Comment by bekah-stephens** [comment]
Figma mocks: https://www.figma.com/design/VMEX8Xg2nzhBX8rfBx53jp/branch/raktQs3QsxK8IDzPhKThmW/PatternFly-6%3A-Components?m=auto&node-id=3-35&t=7JGjC1c3kBnDTw1w-1
